ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/plugin.py

```diff
@@ -50,7 +50,9 @@ def get_auth_functions(self):
 
     def get_helpers(self):
         return {'service_permission_application_url': helpers.service_permission_application_url,
-                'service_permission_applications_enabled': helpers.service_permission_applications_enabled}
+                'service_permission_applications_enabled': helpers.service_permission_applications_enabled,
+                'max_resource_size': toolkit.config.get('ckan.max_resource_size')
```

Comment:
This is not really needed as `get_helpers` return values override functions with the same names. It should be enough to define it in a single place even though it causes a dependancy.